openlibrary/core/lending.py

```diff
@@ -50,7 +50,10 @@ def setup(config):
     global config_content_server, config_loanstatus_url, config_ia_access_secret, config_bookreader_host, config_ia_ol_shared_key
 
     if config.get("content_server"):
-        config_content_server = ContentServer(config.get("content_server"))
+        try:
+            config_content_server = ContentServer(config.get("content_server"))
+        except:
```

Comment:
I am testing and bulletproofing from the outside in. For now, the default value of None at the start of the file will suffice. Agreed that deeper in, this None value may (should) choke. I am counting on that to provide an indication of where next to address.